src/bors/handlers/refresh.rs

```diff
@@ -10,12 +10,21 @@ pub async fn refresh_repository<Client: RepositoryClient>(
     repo: &mut RepositoryState<Client>,
     db: &dyn DbClient,
 ) -> anyhow::Result<()> {
-    let timeout = repo.config.timeout;
+    cancel_timed_out_builds(repo, db).await?;
```

Comment:
We should do these two things always, i.e. if cancelling somehow fails, we should still try to also reload permissions.